### ISSUE TITLE
Spotify-qt 3.10 win64 heavy on resources, 3.8 not

### ISSUE THREAD

**Issue by JustMyGithub** [kraxarn/spotify-qt#228]
### System information
Laptop
Win 10 Pro x64 b19044
i7-8850H

PC
Win 10 Pro x64 b19044
Ryzen5 2600

### Detailed information
3.10 has heavy resource usage on Windows 10 x64. 8% CPU usage (= one of 6*2 SMT cores) and 500MB RAM compared to 3.8 with 20 MB RAM and close to no CPU usage

### What happens?
It is hardly possible to use Spotify-qt at all

**Comment by kraxarn** [comment]
Assuming v3.9 also doesn't have any issues, do you happen to have a lot of liked tracks?

**Comment by JustMyGithub** [comment]
If a few 1000s is a lot then yes.

Not sure why I did not test 3.9 before. 3.9 seems to do the same when "liked tracks" is chosen (GUI locked up, 1GB RAM & 8% CPU for minutes) but finaly succeeds showing the liked tracks at ~130MB RAM.
If I close 3.9 and open 3.10, liked tracks are shown (likely cached by 3.9?), but still 3.10 does put heavy load on one core (reloads the list as it may have changed?) and does for example take about 10 seconds to react when a track in the list is clicked on. I saw a peak at about 9 GB RAM.

In fact, 3.8 does behave similar when liked tracks are chosen, but it seems to give up a some point (does no longer utilize one core but does not show the liked tracks either, not even an incomplete list)
